src/manifest.json

```diff
@@ -11,11 +11,16 @@
     {
       "matches": ["https://github.com/*"],
       "run_at": "document_idle",
-      "css": ["github-patcher.css"],
```

Comment:
It seems time lag to hide stars after removing this static CSS.
So, add it again but set `visibility: hidden` instead if display none.
This aims to promise hiding the counter with broken layout. And it immediately be hidden by scriptiong.insertCSS and `display: none`